src/components/Counter.js

```diff
@@ -20,20 +24,20 @@ class Counter extends Component {
         return (
             <p>
                 Clicked: {this.props.count} times
-                <button onClick={() => /* Fill me in */ }>
+                <button onClick={() => this.props.incrementHandler(this.props.count) }>
```

Comment:
Passing the current count from Redux store into the action is a little bit redundant, since the reducer already has access to the store and knows what the current count is. But the way you have it set up works!